<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="11766" started-at="2021-05-28T06:40:13 IST" finished-at="2021-05-28T06:40:25 IST">
    <groups>
    </groups>
    <test name="MyTestCases" duration-ms="11766" started-at="2021-05-28T06:40:13 IST" finished-at="2021-05-28T06:40:25 IST">
      <class name="org.test.ng.mypackage.seleniumtestng">
        <test-method status="PASS" signature="precondition()[pri:0, instance:org.test.ng.mypackage.seleniumtestng@23faf8f2]" name="precondition" is-config="true" duration-ms="2219" started-at="2021-05-28T06:40:13 IST" finished-at="2021-05-28T06:40:15 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="openurl()[pri:1, instance:org.test.ng.mypackage.seleniumtestng@23faf8f2]" name="openurl" duration-ms="9158" started-at="2021-05-28T06:40:16 IST" finished-at="2021-05-28T06:40:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openurl -->
        <test-method status="PASS" signature="countlinks()[pri:2, instance:org.test.ng.mypackage.seleniumtestng@23faf8f2]" name="countlinks" duration-ms="94" started-at="2021-05-28T06:40:25 IST" depends-on-methods="org.test.ng.mypackage.seleniumtestng.openurl" finished-at="2021-05-28T06:40:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- countlinks -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:org.test.ng.mypackage.seleniumtestng@23faf8f2]" name="afterTest" is-config="true" duration-ms="155" started-at="2021-05-28T06:40:25 IST" finished-at="2021-05-28T06:40:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- org.test.ng.mypackage.seleniumtestng -->
      <class name="org.test.ng.mypackage.Mydemo">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="beforeSuite" is-config="true" duration-ms="8" started-at="2021-05-28T06:40:13 IST" finished-at="2021-05-28T06:40:13 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="beforeTest" is-config="true" duration-ms="1" started-at="2021-05-28T06:40:13 IST" finished-at="2021-05-28T06:40:13 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="beforeClass" is-config="true" duration-ms="1" started-at="2021-05-28T06:40:15 IST" finished-at="2021-05-28T06:40:15 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2021-05-28T06:40:16 IST" finished-at="2021-05-28T06:40:16 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="mytestcase()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="mytestcase" duration-ms="20" started-at="2021-05-28T06:40:16 IST" finished-at="2021-05-28T06:40:16 IST">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[A]> but was:<[actuals]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[A]> but was:<[actuals]>
at org.junit.Assert.assertEquals(Assert.java:115)
at org.junit.Assert.assertEquals(Assert.java:144)
at org.test.ng.mypackage.Mydemo.mytestcase(Mydemo.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mytestcase -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="afterMethod" is-config="true" duration-ms="1" started-at="2021-05-28T06:40:16 IST" finished-at="2021-05-28T06:40:16 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="myothertestcase()[pri:1, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="myothertestcase" duration-ms="0" started-at="2021-05-28T06:40:16 IST" depends-on-methods="org.test.ng.mypackage.Mydemo.mytestcase" finished-at="2021-05-28T06:40:16 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Mydemo.myothertestcase()[pri:1, instance:org.test.ng.mypackage.Mydemo@d44fc21] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Mydemo.myothertestcase()[pri:1, instance:org.test.ng.mypackage.Mydemo@d44fc21] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myothertestcase -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="afterClass" is-config="true" duration-ms="0" started-at="2021-05-28T06:40:16 IST" finished-at="2021-05-28T06:40:16 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="afterTest" is-config="true" duration-ms="1" started-at="2021-05-28T06:40:25 IST" finished-at="2021-05-28T06:40:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:org.test.ng.mypackage.Mydemo@d44fc21]" name="afterSuite" is-config="true" duration-ms="0" started-at="2021-05-28T06:40:25 IST" finished-at="2021-05-28T06:40:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- org.test.ng.mypackage.Mydemo -->
    </test> <!-- MyTestCases -->
  </suite> <!-- Suite -->
</testng-results>
